esp-hal/src/soc/esp32c6/mod.rs

```diff
@@ -60,10 +60,10 @@ pub(crate) mod constants {
     /// The clock frequency for the Parallel IO peripheral in Hertz.
     pub const PARL_IO_SCLK: u32 = 240_000_000;
 
-    /// The lower address boundary for system DRAM.
-    pub const SOC_DRAM_LOW: usize = 0x4080_0000;
-    /// The upper address boundary for system DRAM.
-    pub const SOC_DRAM_HIGH: usize = 0x4088_0000;
+    /// The lower bound of the system's DRAM (Data RAM) address space.
```

Comment:
maybe all of these constants should come from esp-metadata some day

Reply:
I like the idea 👍